app/views/components/ilequery/response_ducr_summary.scala.html

```diff
@@ -62,6 +62,14 @@ <h2 class="govuk-heading-m">@messages("ileQueryResponse.details")</h2>
                 value = Value(
                     content = info.entryStatus.flatMap(_.ics).map(converter.inputCustomsStatus(_)).getOrElse(Empty)
                 )
+            ),
+            SummaryListRow(
+                key = Key(
+                    content = Text(messages("ileQueryResponse.details.MRN"))
+                ),
+                value = Value(
+                    content = HtmlContent(info.declarationId)
```

Comment:
Are you sure that is not our internal id ? MRN ?

Reply:
Yes, it is stated in the ticket, and to double-check, I asked Ian as well.